directdemod/merger/config.py

```diff
@@ -0,0 +1,4 @@
+### Configuration file for ImageMerger class
```

Comment:
We try to place all constants in constants.py, so that this user specific file is the only one that the user needs to customize. Whenever the user upgrades to a new release a replacement of just this file would suffice. Hence I would recommend you to move constants in config.py to constants.py.

Reply:
@7andahalf Thanks for reply! I will move these to constants.py. It will really make more sense then.